saleor/order/models.py

```diff
@@ -65,11 +65,9 @@ class Order(models.Model):
         net_field='shipping_price_net', gross_field='shipping_price_gross')
     token = models.CharField(max_length=36, unique=True)
     total_net = MoneyField(
-        currency=settings.DEFAULT_CURRENCY, max_digits=12, decimal_places=2,
-        blank=True, null=True)
+        currency=settings.DEFAULT_CURRENCY, max_digits=12, decimal_places=2)
```

Comment:
Wouldn't `default=0` do the trick here? We're setting zero price anyway in all places where an order is created.

Reply:
You'll actually need `default=Money(0, settings.DEFAULT_CURRENCY)` AFAIK, but thats beyond the contest ;)

I've went with explicit total because I wanted those saving `Order` to database to be conscious about what total gets saved, but then only place where this matters in our codebase is checkout where we are taking total from cart, so that didn't work out in the first place.